public/Anirudh Anup

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Anirudh Anup",
+  "bio": "I am Anirudh Anup, A passionate 13 year old developer from India, I am always open to collaborating on projects",
+  "avatar": "https://github.com/kaiwalyakoparkar.png",
```

Comment:
- Please check the Avatar url I seems not to be yours.

Reply:
opps thanks